rmgpy/cantherm/thermo.py

```diff
@@ -248,5 +247,10 @@ def plot(self, outputDirectory):
         pylab.ylabel('Gibbs free energy (kcal/mol)')
 
         fig.subplots_adjust(left=0.10, bottom=0.08, right=0.95, top=0.95, wspace=0.35, hspace=0.20)
-        pylab.savefig(os.path.join(outputDirectory, 'thermo.pdf'))
+
+        if not os.path.exists('plots'):
+            os.mkdir('plots')
+        valid_chars = "-_.()<=> %s%s" % (string.ascii_letters, string.digits)
+        filename = 'plots/' + ''.join(c for c in self.species.label if c in valid_chars) + '.pdf'
```

Comment:
This line just manipulates a string. I do use `os.path.join` in the next line

Reply:
Yeah, you join `outputDirectory` and `filename`, but you should also use `os.path.join` to join `plots` and whatever follows instead of doing `'plots/' + ...`